.eslintrc.json

```diff
@@ -34,7 +34,8 @@
         ]
       }
     ],
-    "simple-import-sort/exports": "error"
+    "simple-import-sort/exports": "error",
+    "import/no-relative-parent-imports": "error"
```

Comment:
Can we set this to never but add overrides for certain files? I think `.stories.tsx`, `.element.tsx` and `index.ts` are the only files that should have relative imports.

Reply:
"Can we set this to never even for children?"

Was there a typo here? Children of whom?

Reply:
file specific overrides: [ef0f9c9](https://github.com/seamapi/react/pull/464/commits/ef0f9c919369aacefa82dd1ffe42c42fac5da389)

Reply:
Small misunderstanding. Safe to revert ef0f9c919369aacefa82dd1ffe42c42fac5da389 since those are all child imports (non parents). The overrides are not needed (we never want relative parent imports).